.github/workflows/unittest.yml

```diff
@@ -23,6 +23,10 @@ jobs:
           export CPLUS_INCLUDE_PATH=/usr/include/gdal
           export C_INCLUDE_PATH=/usr/include/gdal
           pip install .
+          pip install coverage
 
       - name: Run All Unittest
-        run: python -m unittest discover terragraph_planner -b
+        run: coverage run -m unittest
```

Comment:
Are you replacing the standard unit test run with the coverage run? It seems we might want to have both.

Reply:
Coverage run will run all the tests too. If we have both, we will need double time.

Reply:
What happens to coverage if a unit test fails? And what happens if the unit test is fixed -- does the coverage report get updated? One question is if coverage should *only* be run once the unit tests all pass.